public/creators-landing/src/locale/en.js

```diff
@@ -62,7 +62,7 @@ export default {
   signoff: {
     btn: "Sign up now",
     btnHref: "/sign-up",
-    headlineOne: "Join ",
+    headlineOne: "Join over ",
```

Comment:
Can we remove headlineTwo and change this to be 

```yaml
headline: "Join over {count} content creators and publishers earning more from Brave Rewards",
```

And then in the code located in the `signoff/index.js` page we can have something similar to what the [example in the react-intl](https://github.com/formatjs/react-intl#example) contains

```jsx
<Heading alignSelf='center' level='4' textAlign='center'>
  <FormattedMessage id="signoff.headline" values={{count: <b>500,000</b>}}/>
</Heading>
```

Reply:
One possible hiccup is that the translated text (at least from AWS) doesn't even have the count formatted as we would in English:

> 50万人以上のコンテンツクリエイターや出版社に参加して、Brave Rewardsからより多くの収入を得ましょう

Wondering if there's value in not even interpolating the count at all.

Reply:
Possibly! I would like to introduce a PR that use calculates the number based on [totals endpoint](https://publishers-distro.basicattentiontoken.org/api/v3/public/channels/totals) so we no longer have to manually update it. 

I'm going to start a chat on Slack with you and Ryotaro and see what he thinks. 😄

Reply:
I've updated as suggested (with a few slight changes).

Below is what it looks like in Japanese. Confirming with Ryotoro in Slack.

<img width="1713" alt="Screen Shot 2020-04-09 at 3 41 21 PM" src="https://user-images.githubusercontent.com/1199820/78934430-ea130280-7a78-11ea-993f-ae0eedb4530a.png">